src/index.html

```diff
@@ -1,13 +1,457 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>MyBiKE</title>
+    <link
+      rel="icon"
+      type="image/x-icon"
+      href="./images/favicon.svg"
+    >
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="header">
+      <div class="header__content">
+        <div class="header__nav">
+          <a href="#">
+            <img
+              class="logo__image"
+              src="./images/MyBiKE--black.svg"
+              alt="logo"
+            >
+          </a>
+
+          <div class="header__info">
+            <a
+              href="tel: +1 234 5555-55-55"
+              class="icon header__contact"
+            >
+            </a>
+
+            <a
+              href="#menu"
+              class="icon header__menu-opener"
+            >
+            </a>
```

Comment:
Your menu icon is cropped
![image](https://user-images.githubusercontent.com/37161315/197524680-92fbe1bf-2965-49b6-b30d-71c12e58f69b.png)